apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -46,6 +46,7 @@ import {
   getCurrentTimestamp,
   hasFilledOutModelCard,
 } from './utils';
+import {validateModelId} from '../views/modelCustomization/utils';
```

Comment:
Nice! Just confirming we need this for AI customizations that have been saved in the past and we don't need this check in `saveAiCustomization` because we're assuming that users can only selected from updated `modelDescriptions.json`. Is that correct?

Reply:
Yep, I think so - this is specifically for when we change the model descriptions JSON and levels or student code have old invalid IDs saved.